server/components/email/index.js

```diff
@@ -13,9 +13,11 @@ var eventReminderTemplate = fs.readFileSync(path.join(__dirname,"templates","eve
 var emailClient;
 switch(config.emailService){
   case "SMTP":
+    console.log("Setting email to SMTP")
     emailClient = require("./smtp");
     break;
   case "SENDGRID":
+    console.log("Setting email to SENDGRID")
```

Comment:
you should make sure it only logs with a flag? Or do you think this is desirable? I would imagine with 300 students this would fill up stdout very quickly